### PR TITLE
EOS-19779: Update mini provisioner template files

### PR DESCRIPTION
**Pull request by akash2144**
Signed-off-by: Akash Dudhane <akash.a.dudhane@seagate.com>

## Problem Statement
<pre>
  <code>
  Story Ref (if any):
    Add "public_fqdn" key in mini provisioner template files.
  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
  Yes/No
  </code>
</pre>
## Problem Description
<pre>
  <code>
    Add "public_fqdn" in mini provisioner template files.
  </code>
</pre>
## Solution
<pre>
  <code>
    Updated template files : Added "public_fqdn" key in template files.
  </code>
</pre>
## Unit Test Cases
<pre>
  <code>
    Tested and confirmed that all mini provisioner commands works fine with updated template files.
  </code>
</pre>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Review by indrajitzagade — APPROVED** [review]

**Comment by indrajitzagade** [comment]
@akash2144 Please update the PR template

**Review by indrajitzagade — CHANGES_REQUESTED** [review]
Please update the PR template

**Comment by akash2144** [comment]
> Please update the PR template

Updated.

**Review by indrajitzagade — APPROVED** [review]

File: conf/mini_provisioner/v2/ha.cleanup.tmpl.1-node [hunk 1]
```diff
@@ -22,5 +22,6 @@ server_node:
     site_id: TMPL_SITE_ID
     rack_id: TMPL_RACK_ID
     storage_set_id: TMPL_STORAGE_SET_ID
+    host_id: TMPL_HOST_ID
```
Comment (ajaysrivas) [inline conf/mini_provisioner/v2/ha.cleanup.tmpl.1-node:25]: what is this key? Why is it required? There is nothing like host_id in mini-provisioner confstore - https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/85000402/Mini-Provisioner+ConfStore+Keys

Comment (indrajitzagade) [reply conf/mini_provisioner/v2/ha.cleanup.tmpl.1-node:25]: Yeah good catch, it has to be "server_node>{machine_id}>network>management>public_fqdn

**Review by indrajitzagade — CHANGES_REQUESTED** [review]
Please fix the key as @ajaysrivas mentioned in this comment

File: conf/mini_provisioner/v2/ha.cleanup.tmpl.1-node (hunk 1 above)
Comment (ajay-paratmandali) [reply conf/mini_provisioner/v2/ha.cleanup.tmpl.1-node:25]: are we using it in mini provision ha ?

if yes 
please create pr to update dev_env for main branch
https://github.com/Seagate/cortx-ha/blob/main/jenkins/dev_env/conf/example_config.json

Comment (akash2144) [reply conf/mini_provisioner/v2/ha.cleanup.tmpl.1-node]: @ajay-paratmandali As i changed "host_id" to "public_fqdn", now there is no need to update dev_env .
"public_fqdn" is already present there.

**Review by ajay-paratmandali — APPROVED** [review]

**Review by ajaysrivas — APPROVED** [review]

**Review by indrajitzagade — APPROVED** [review]

**Pull request merged**